docs/_static/spec/server.yaml

```diff
@@ -0,0 +1,1517 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PredictResult'
+              example:
+                PredictResult:
+                  $ref: '#/components/examples/PredictResult'
+        400:
+          $ref: '#/components/responses/400BadRequest'
+        401:
+          $ref: '#/components/responses/401NotAuthenticated'
+        403:
+          $ref: '#/components/responses/403NotAuthorized'
+        500:
+          $ref: '#/components/responses/500ServerError'
+
+  /models/parse:
+    post:
+      security:
+      - TokenAuth: []
+      - JWT: []
+      tags:
+      - Model
+      summary: Parse a message using the stacked or NLU model.
+      description: >-
+        Predicts the intent and entities of the message
+        posted to this endpoint. No messages will be stored
+        to a conversation and no action will be run. This will
+        just retrieve the NLU parse results.
+      parameters:
+      - $ref: '#/components/parameters/model_name'
+      - $ref: '#/components/parameters/model_type'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                text:
+                  type: string
+                  description: Message to be parsed
+            example:
+              text: "Hello, I am Rasa!"
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ParseData'
+        400:
+          $ref: '#/components/responses/400BadRequest'
+        401:
+          $ref: '#/components/responses/401NotAuthenticated'
+        403:
+          $ref: '#/components/responses/403NotAuthorized'
+        500:
+          $ref: '#/components/responses/500ServerError'
+
+  /models:
+    put:
+      security:
+      - TokenAuth: []
+      - JWT: []
+      tags:
+      - Model
+      summary: Replace a trained model
+      description: >-
+        Upload a trained model to the server. The server
+        will unzip the uploaded file and try to load the model.
+        This will replace the previously loaded model.
+      parameters:
+      - $ref: '#/components/parameters/model_type'
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              properties:
+                model:
+                  description: Zipped model file
+                  type: string
+                  format: binary
+      responses:
+        204:
+          description: Could successfully replace model.
+        400:
+          $ref: '#/components/responses/400BadRequest'
+        401:
+          $ref: '#/components/responses/401NotAuthenticated'
+        403:
+          $ref: '#/components/responses/403NotAuthorized'
+        500:
+          $ref: '#/components/responses/500ServerError'
+
+    delete:
```

Comment:
what does this do, unload the current model and set the currently loaded model to `None`?

Reply:
Yes. Not sure if the endpoint is still needed.